src/content/public goods funding.md

```diff
@@ -0,0 +1,281 @@
+
+### Incentive alignment {#incentive-alignment} 
+
+Public goods markets on Ethereum are designing better incentives to encourage greater public investment in digital public goods. Part of this approach involves designing better systems for measuring public opinion on what public goods deserve funding. 
+
+For example, some Ethereum-based public goods markets use a quadratic voting system to control the allocation of funds to organizations building digital public goods. [Quadratic voting](https://vitalik.ca/general/2019/12/07/quadratic.html) is considered ideal because it strikes a balance between one-person-one-vote and one-dollar-one-vote systems. 
+
+Below is an overview of how the combination of quadratic voting with public fundraising (**quadratic funding**) works:
+
+1. Individuals and organizations donate funds to a  "matching pool" at the beginning of a funding round. These funds are distributed to different public goods projects at the end of the round based on their weighted share of votes. 
+
+2. Anyone can "vote" for a project by donating some amount to it during the funding round. The influence of each voter is calculated as the square root of the amount donated. If Alice donates $100, her influence on the outcome equals that of ten people who contribute $1. 
+
+3. Every donation towards a project is matched with a corresponding amount of funding from the official matching pool. Instead of being matched 1:1, funds are matched according to the [quadratic funding formula](https://wtfisqf.com/?grant=10&grant=5,5&grant=2,2,2,2,2&grant=1,1,1,1,1,1,1,1,1,1&match=1000). 
+
+The formula works by adding the square roots of each contribution, squaring the result, and subtracting the original amount donated from it. The final figure represents the matched funding, i.e., how much money each project receives from the matching pool. We can explain this concept with an example:
+
+Imagine two parks are planned for development, with the teams building each park (we call them `park A` and `park B`) requesting for funding. Ten people agree to donate $25 each to build `park A`, while two affluent investors donate $180 each to `park B`. 
+
+The table below shows how much each park will receive under a quadratic funding scheme:
+
+![A diagram showing showing funds received during a quadratic funding round](./quadratic-funding-table.png)
+
+We can easily calculate the final amount received by each project using the quadratic formula: 
+
+1. `Park A`: Adding the square roots of each donor's contribution ($5) gives $50. Subtracting the total amount contributed ($250) from the square of $50 ($2,500) gives $2,250—the matched funds. 
+
+2. `Park B`: Adding the square roots of each donor's contribution ($20) gives $40. Subtracting the total amount contributed ($800) from the square of $40 ($1,600) gives $800—the matched funds. 
+
+#### Why is quadratic funding ideal? {#advantages-of-quadratic-funding} 
```

Comment:
"ideal" might be a bit generous - certainly it is favourable compared to traditional funding mechanisms but it is also inherently vulnerable to Sybils whose influence are quadratically scaled. I'd favour using the page-tag wording in the title (i.e. change title to "advantages of quadratic funding")

Reply:
Changed this per your suggestion.